### DISCUSSION TITLE
I was able to put the sensor watch pcb into a Casio W-59 (module 590) which has 50m water resistance.

### DISCUSSION THREAD

**Discussion by sprayed** [joeycastillo/Sensor-Watch#320] [Show and tell]
The module 590 seems identical to module 593 in every regard, except cosmetically the size and position of the LCD segments. The installation process is exactly the same as installing on the f91w. This watch presents an excellent alternative to the f91w for improved water resistance. I took a video of the installation process, but wanted to share the news here first.

**Comment by BrianBinFL** [comment]
I'd be happy to see your installation video as I'm sure would others.

**Reply by sprayed** [reply]
I have not yet edited it because it is not any different than installing it in a f91w.

**Comment by sidrichmill** [comment]
Did you use an LCD from a F91W or is the LCD for the W-59 pin compatible? That would open up more options for compatibility!

**Reply by sprayed** [reply]
It's 100% compatible, including the LCD pins, just like using a f91w.

**Comment by 814d3** [comment]
I'm a bit confused. It seems there are two kind of W-59 LCD?
![image](https://github.com/joeycastillo/Sensor-Watch/assets/82269787/1fc0c621-08ba-4726-a731-6e68defff9a5)
I like the one with the small frame around the current day. Is this the one you test?

Edit:
The one with the small frame seems to be the former version of the LCD. The ones sold/produced today don't have it (my guess!).
But are the LCDs of F-91W and W-59 really compatible? It's obvious, that they differ in the "LAP" and "SPLIT" display.
![image](https://github.com/joeycastillo/Sensor-Watch/assets/82269787/c343ea0e-40f6-44f3-8229-0ad14c08964e)

Edit2:
Got an W-59 now and can confirm that the LCD works as expected with Split instead of LAP indicator.

**Reply by dvelazquez** [reply]
If I knew before I would have bought the water resistant version.
I might buy it for a gift though

thanks for checking
